c_glib/arrow-glib/basic-array.h

```diff
@@ -890,6 +890,10 @@ GARROW_AVAILABLE_IN_ALL
 GArrowDecimal256 *
 garrow_decimal256_array_get_value(GArrowDecimal256Array *array, gint64 i);
 
+GARROW_AVAILABLE_IN_20_0
+gboolean
+garrow_array_validate(GArrowArray *array, GError **error);
```

Comment:
Could you move this to the `garrow_array_*()` block?
https://github.com/apache/arrow/blob/f8f17a58adb6b5fe4fed6c13d7d3b2e64b4a9acd/c_glib/arrow-glib/basic-array.h#L128

Reply:
Thanks! moved.